drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#include <dlfcn.h>
```

Comment:
System includes should come last, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
More than for cosmetic reasons BTW, non-Unix builds fail now :)
https://ci.appveyor.com/project/akien-mga/godot/build/1.0.13416